spec/controllers/spree/admin/users_controller_spec.rb

```diff
@@ -38,4 +38,21 @@
       expect(response).to redirect_to('/unauthorized')
     end
   end
+
+  describe "#accept_terms_of_services" do
+    let(:user) { create(:user) }
+
+    before do
+      allow(controller).to receive_messages spree_current_user: user
+      user.spree_roles << Spree::Role.find_or_create_by(name: 'admin')
```

Comment:
This means super-admin. The users controller is only for super admin and not for regular enterprise users. So I think that this won't work for most users because they don't have permission on the controller.